install/kubernetes/cilium/values.yaml

```diff
@@ -359,3 +359,42 @@ global:
   # packets affected by policies will not be dropped. Policy related
   # decisions can be checked via the poicy verdict messages.
   policyAuditMode: false
+
+  # hubble configures Hubble.
+  hubble:
+    # List of unix domain socket paths to listen to, for example:
+    #
+    #   listenAddresses:
+    #   - "unix:///var/run/cilium/hubble.sock"
+    #
+    # You can specify the list of metrics from the helm CLI:
+    #
+    #   --set global.hubble.listenAddresses={unix:///var/run/cilium}
+    #
+    # Hubble is disabled if the list is empty.
+    listenAddresses: []
+    # Buffer size of the channel Hubble uses to receive monitor events. Defaults to 128.
+    eventQueueSize: ~
```

Comment:
Apparently I missed this in the other PR, but 128 (which is taken from stand-alone) is way to too small for embedded mode, Hubble will drop most flows with this setting (https://github.com/cilium/hubble/issues/161).

This means we need to the bump the default for `option.HubbleEventQueueSize` here as well:
https://github.com/cilium/cilium/blob/8d28ab13be7d2aeca75702b89445a0d26c4815ca/daemon/daemon_main.go#L738

I propose we use the same value as we use for `MonitorQueueSize`:
https://github.com/cilium/cilium/blob/8d28ab13be7d2aeca75702b89445a0d26c4815ca/pkg/option/config.go#L2166-L2171

Reply:
Let's fix it in a separate PR so it's documented properly why we bumped it.

Reply:
https://github.com/cilium/cilium/pull/10488 cc @gandro